LayoutTests/imported/w3c/web-platform-tests/css/css-sizing/contain-intrinsic-size/auto-012-expected.txt

```diff
@@ -1,9 +1,9 @@
 
-PASS Sizing normally
+FAIL Sizing normally assert_equals: clientWidth expected 50 but got 100
```

Comment:
@mrobinson Do you know how this subtest passes in Gecko without the sync intersection observer (or is there maybe one that I missed?)?

Reply:
I have forgotten a lot of things, but my intuition is that this is passing due to the `nextRendering()` await on two calls to `requestAnimationFrame()`. I suspect that is why this can pass in Gecko.